src/utils/scores.test.ts

```diff
@@ -120,25 +120,45 @@ describe('score tests', () => {
     ORDER BY num_comments DESC
     ;`;
 
-    const discussionCommentersQuery = `
+    const githubCommentersQuery = `
     SELECT
-      discussions.username as username,
-      COUNT(discussions.username) as num_comments,
+      issues.username as username,
+      COUNT(issues.username) as num_comments,
     FROM
-      \`open_source.github_events\` AS discussions
+      \`open_source.github_events\` AS issues
```

Comment:
nit: `issues` -> `comments`? That seems to be what we're actually grabbing, if the `.type` values we filter for are anything to go by.

Reply:
done